src/reducers/index.js

```diff
@@ -57,9 +57,27 @@ export const removeResource = (state, action) => {
   return validate(newState)
 }
 
+// shameless green -- nearly duplicates setRetrieveError()
+export const setPublishError = (state, action) => {
+  // Use safe navigation for dynamic properties: https://github.com/tc39/proposal-optional-chaining#syntax
+  const reason = action.payload?.reason
```

Comment:
I don't think you need safe nav hear because `action.payload` is guaranteed to be present.

Reply:
It's because of this:
![image](https://user-images.githubusercontent.com/96775/62333469-69771d80-b478-11e9-8b70-3ceb0e1f551f.png)

Reply:
I will try adding empty payload to test.  :-)

Reply:
You are correct, sir!